pages/api/import/savvycal.ts

```diff
@@ -0,0 +1,78 @@
+import { PrismaClient } from "@prisma/client";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { getSession } from "@lib/auth";
+
+const prisma = new PrismaClient();
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const session = await getSession({ req });
+  const authenticatedUser = await prisma.user.findFirst({
+    rejectOnNotFound: true,
+    where: {
+      id: session?.user.id,
+    },
+    select: {
+      id: true,
+    },
+  });
+  if (req.method == "POST") {
```

Comment:
Let's stick to strict equality

```suggestion
  if (req.method === "POST") {
```